<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AQDevelWidget</class>
 <widget class="QWidget" name="AQDevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_38">
     <property name="horizontalSpacing">
      <number>50</number>
     </property>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_1871">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_510">
         <item>
          <widget class="QLabel" name="label_155">
           <property name="text">
            <string>Focal Length in mtrs.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_20">
           <property name="text">
            <string>0.01</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_810">
         <item>
          <widget class="QLabel" name="label_138">
           <property name="text">
            <string>Yaw Offset [radians]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_19">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_136">
           <property name="text">
            <string>Pitch Offset [radians]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_18">
           <property name="text">
            <string>0.7854</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1010">
         <item>
          <widget class="QLabel" name="label_135">
           <property name="text">
            <string>Pixel Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_171">
           <property name="text">
            <string>7e-6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1110">
         <item>
          <widget class="QLabel" name="label_156">
           <property name="text">
            <string>Camera number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_21">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_189">
           <item>
            <widget class="QLabel" name="label_158">
             <property name="text">
              <string>X-min changes [pix]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_22">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1881">
         <item>
          <widget class="QLabel" name="label_159">
           <property name="text">
            <string>Y-min changes [pix]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_23">
           <property name="text">
            <string>15</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_24">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ObjectTracking_File">
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ObjectTracking">
         <property name="text">
          <string>Start Object Tracking</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_17">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_213">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_212">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Heading Absolut in Â° (0.0-359.9)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_13"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_202">
         <item>
          <widget class="QLabel" name="label_111">
           <property name="text">
            <string>Free</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_201">
         <item>
          <widget class="QLabel" name="label_132">
           <property name="text">
            <string>Free</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_16"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_192">
         <item>
          <widget class="QLabel" name="label_134">
           <property name="text">
            <string>Free</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_dev1">
         <property name="text">
          <string>Send To Aq</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
